src/libraries/System.Runtime/tests/System.Globalization.Tests/CultureInfo/CultureInfoCtor.cs

```diff
@@ -388,6 +388,18 @@ public static IEnumerable<object[]> Ctor_String_TestData()
             }
         }
 
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsIcuGlobalization))]
+        [InlineData("und-us", "und-US", "und-US")]
+        [InlineData("und-us_tradnl", "und-US", "und-US_tradnl")]
+        [InlineData("und-es-u-co-phoneb", "und-ES", "und-ES_phoneb")]
+        [InlineData("und-es-t-something", "und-ES", "und-ES")]
```

Comment:
Is it possible to add a test for `und-fonipa`? That's a common tag for IPA, referenced on Wikipedia for example: https://en.wikipedia.org/wiki/International_Phonetic_Alphabet#IETF_language_tags

Reply:
@xadxura may help with this too?

Reply:
@xadxura I am seeing Windows don't allow `und-Latn-fonipa` when calling `GetLocaleInfoEx`. Am I missing something? should `und-Latn-fonipa` need to get registered on Windows before using this tag?

Reply:
Thanks for the updates @tarekgh and @xadxura. Duh, I knew that Windows needed `Latn` but just plain forgot when asking here. Sorry for the side-track. But it sounds like `und-Latn-fonipa` still has an open question.

Reply:
Yes, custom BCP-47 tags must be registered on the system first.